movieReviewApp/src/main/java/com/mobiquity/movieReviewApp/domain/accountmanagement/service/UtilityService.java

```diff
@@ -47,24 +48,39 @@ public void sendPasswordForgotLink(UserProfile user) {
         email.setText(
                 "reset link valid for 24 hrs" + "\n"
```

Comment:
This should be 30 minutes instead of 24 hrs, as in you token generation with emailId and password method you use 30 minutes